agent/engine/dockerclient/versionsupport_windows.go

```diff
@@ -1,3 +1,4 @@
+// +build windows
```

Comment:
I have no idea why git thinks you moved a file here...did you intentionally get rid of `dummy_test.go`?